### ISSUE TITLE
preferred citation?

### ISSUE THREAD

**Issue by dylancraven** [MoBiodiv/mobr#164]
Do you have a preferred citation for mobr or is it too soon?

**Comment by dmcglinn** [comment]
Hey @dylancraven! I'm glad the package is proving useful for you, `citation('mobr')` returns the proper citation info for the package:

```r
 To cite package ‘mobr’ in publications use:

  Xiao Xiao, Daniel McGlinn and Felix May (2017). mobr:
  Measurment of Biodiveristy in R. R package version 1.0.
```
But I'm glad you asked because there is a typo in that I'll fix real quick. Also we have a manuscript circulating with coauthors that describes the package. We'll let folks know when we release a preprint of that paper.

**Comment by dmcglinn** [comment]
solved in e7c2c894268e49f201e47f8e2e6d49506cc89879
